src/lib.rs

```diff
@@ -45,149 +45,124 @@
 
 extern crate rusoto_core;
 
-extern crate chrono;
-extern crate hyper;
-#[macro_use]
-extern crate log;
-extern crate md5;
-extern crate rustc_serialize;
-extern crate serde;
-// Only bring in serde_derive for services that use it.
-#[cfg(feature = "needs_serde_derive")]
-#[macro_use]
-extern crate serde_derive;
-extern crate serde_json;
-extern crate xml;
-
 pub use rusoto_core::*;
 
-pub use rusoto_core::credential::{AwsCredentials, ChainProvider, ContainerProvider, CredentialsError,
-                                  EnvironmentProvider, InstanceMetadataProvider, ProfileProvider,
-                                  ProvideAwsCredentials, DefaultCredentialsProvider,
-                                  DefaultCredentialsProviderSync, claims,
-                                  AutoRefreshingProviderSync, AutoRefreshingProvider,
-                                  BaseAutoRefreshingProvider};
-
-#[cfg(test)]
-mod mock;
-
 #[cfg(feature = "acm")]
-pub mod acm;
+pub extern crate rusoto_acm as acm;
```

Comment:
This gets us into the larger discussion of "how will these crates all get packaged once this is landed", but that's a discussion for another PR.